fenix/app/src/main/java/org/mozilla/fenix/addons/AddonsManagementView.kt

```diff
@@ -36,9 +35,7 @@ class AddonsManagementView(
         showNotYetSupportedAddonFragment(unsupportedAddons)
     }
 
-    override fun shouldShowFindMoreAddonsButton(): Boolean {
-        return !Config.channel.isRelease
-    }
+    override fun shouldShowFindMoreAddonsButton(): Boolean = true
```

Comment:
If we are going to keep it enable, should we just remove `shouldShowFindMoreAddonsButton`?

Reply:
Should we instead introduce a nimbus feature flag (`true` by default), just to be able to control this functionality via nimbus?

Reply:
> If we are going to keep it enable, should we just remove `shouldShowFindMoreAddonsButton`?

It's part of the public A-C API now (`AddonsManagerAdapterDelegate`) so that's why I kept the override. In Fenix, we want this button but in other apps? I am not sure.

> Should we instead introduce a nimbus feature flag (`true` by default), just to be able to control this functionality via nimbus?

I don't think we need that.

Reply:
Got it 👍🏽 
Thanks for the clarification!